lms/djangoapps/teams/static/teams/js/views/create_team.js

```diff
@@ -0,0 +1,43 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'js/components/header/models/header',
+        'js/components/header/views/header',
+        'teams/js/views/edit_team'],
+       function (Backbone, _, gettext, HeaderModel, HeaderView, TeamEditView) {
+           return Backbone.View.extend({
+               initialize: function(options) {
+
+                   this.headerView = new HeaderView({
+                       model: new HeaderModel({
+                           description: gettext("Create a new team if you can't find existing teams to join, or if you would like to learn with friends you know."),
+                           title: gettext("Create a New Team"),
+                           breadcrumbs: [
+                               {
+                                   title: options.teamParams.topicName,
+                                   url: Backbone.history.location.href.replace(/\/create_new_team\/?/,  '')
```

Comment:
Andy's in-progress [createViewWithHeader](https://github.com/edx/edx-platform/pull/9105/files#diff-b4e47c67e8ac43c1a3ebdfd5eaa30c04R179) should help you here. You're also missing a step in the breadcrumb; currently it is "Algorithms > Create a New Team", but it should be "All Topics > Algorithms > Create a New Team" similar to [this page](http://andy-armstrong.m.sandbox.edx.org/courses/course-v1:AndyA+AA101+1/teams/#teams/fossil-fuels/dead-dinosaurs)